hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -127,6 +127,18 @@
                The width is determined by `RngBusBitSelWidth` parametrization.
                '''
     }
+    { struct:  "logic"
+      type:    "uni"
+      name:    "entropy_src_xht_health_test_window"
+      act:     "req"
+      width:   HealthTestWindowWidth
+      package: ""
+      desc:    '''
+               Provides bit selection information for the raw entropy data.
+               It specifies which specific bit or subset of bits from `entropy_src_xht_bit` should be used.
+               The width is determined by `RngBusBitSelWidth` parametrization.
```

Comment:
This description seems not accurate.

Reply:
Argh. Took me a while to understand; I didn't see the context. of the lines above.